### ISSUE TITLE
Cordova Barcode Scanner plugin not installed in Developer App

### ISSUE THREAD

**Issue by cookieguru** [eclipsesource/tabris-js#1805]
### Problem description

In the latest beta of the developer app, the Cordova Barcode Scanner plugin is not installed.  When tapping the "Scan Barcode" button, the following error is seen:

```plain
TypeError: Cannot read property 'scan' of undefined
  at _scanBarcode (./BarcodeScannerPage.js:31:35)
  at on (./BarcodeScannerPage.js:18:34)
```

### Environment

- Tabris.js version: 3.0.0-beta2
- Device: Various
- OS: Android 8 and 9

**Comment by mpost** [comment]
I can not confirm the issue with the 3.0.0-beta2 release. Anybody else seeing that issue?

**Comment by cookieguru** [comment]
Does this logcat output help any?

```plain
03-04 00:45:38.486 30139 30139 D Tabris  : ERROR TypeError: Cannot read property 'scan' of undefined
03-04 00:45:38.486 30139 30139 D Tabris  :   at _scanBarcode (./BarcodeScannerPage.js:31:35)
03-04 00:45:38.486 30139 30139 D Tabris  :   at on (./BarcodeScannerPage.js:18:34)
03-04 00:45:38.497 30139 30139 E ource.tabris.j: Invalid ID 0x00000650.
03-04 00:45:38.499 30139 30139 E ource.tabris.j: Invalid ID 0x00000650.
03-04 00:45:38.500 30139 30139 W StaticLayout: maxLineHeight should not be -1.  maxLines:2 lineCount:2
03-04 00:45:38.500 30139 30139 I chatty  : uid=10195(com.eclipsesource.tabris.js) identical 8 lines
03-04 00:45:38.500 30139 30139 W StaticLayout: maxLineHeight should not be -1.  maxLines:2 lineCount:2
```

Also, the exact version is 3.0.1-20190228-0553

**Comment by cookieguru** [comment]
Can also confirm that the Camera and Storage permissions are granted

**Comment by mpost** [comment]
We have tried to reproduce the issue to no success. Are you able to reproduce it on other devices as well?

**Comment by cookieguru** [comment]
Able to reproduce on:
* Nexus 5 (Android 6.0.1)
* Samsung Galaxy Tab E (Android 7.1.1)
* Samsung Galaxy Tab A (Android 8.1.0)
* Google Pixel 3 (Android 9)
* iPhone 6 (iOS 12.0)
* iPad mini 4 (iOS 12.1.4)

The Apple devices don't show beta2; specifically it's 3.0.1 (1902251142) Tabris.js 3.0.0

**Comment by mpost** [comment]
That is an exhaustive list thanks. Do you also see the problem with the nightly? Btw... We are now publishing the 3.x nightly android apps to the public Google Play beta track of the tabris.js app.

**Comment by cookieguru** [comment]
Same issue with 3.0.2.-20190307-0907 from the Play Store

**Comment by josiahking** [comment]
Environment

    Tabris.js version: 3.0
    Device: Samsung Galaxy Grand Prime Plus
    OS: Android 6

**Comment by fax1ty** [comment]
```Tabris.js version: 3.0 beta-2
Device: Xiaomi Redmi 5 Plus
OS: Android 8```

**Comment by cookieguru** [comment]
Fixed with 5e6ce550d346fecd84c202882e3c9bbdd4de3616
